pkg/candebug/http_test.go

```diff
@@ -23,9 +25,19 @@ func TestServeMessagesHTTP_Single(t *testing.T) {
 			},
 		})
 	}))
-	res, err := http.Get(ts.URL)
+	c := http.Client{
```

Comment:
Is this for wanting to set 30 second timeout? Is that needed on the test? Wondering whether we can simplify by using the default client instead.

Reply:
The linter complains when you use http.Get without a context. An option would be to disable linter for these cases in tests.

Reply:
Got it. My thinking was to replace this with `http.DefaultClient` for simplicity